prisma/migrations/20230214212125_add_tags_and_refactor_names/migration.sql

```diff
@@ -0,0 +1,117 @@
+  - Added the required column `contactType` to the `ContactMethods` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `organizationId` to the `ContactMethods` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `organizationId` to the `Managers` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `foundationDate` to the `Organization` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `longDescription` to the `Organization` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `numberOfUsers` to the `Organization` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `ownerId` to the `Organization` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `slug` to the `Organization` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `organizationId` to the `Projects` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- DropForeignKey
+ALTER TABLE "ContactMethods" DROP CONSTRAINT "ContactMethods_organization_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Managers" DROP CONSTRAINT "Managers_organization_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Organization" DROP CONSTRAINT "Organization_owner_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Projects" DROP CONSTRAINT "Projects_organization_id_fkey";
+
+-- DropIndex
+DROP INDEX "Organization_owner_id_key";
+
+-- AlterTable
+ALTER TABLE "ContactMethods" DROP COLUMN "contact_link",
+DROP COLUMN "contact_type",
+DROP COLUMN "organization_id",
+ADD COLUMN     "contactLink" TEXT NOT NULL,
+ADD COLUMN     "contactType" TEXT NOT NULL,
+ADD COLUMN     "organizationId" TEXT NOT NULL;
+
+-- AlterTable
+ALTER TABLE "Managers" DROP COLUMN "email_address",
+DROP COLUMN "organization_id",
+ADD COLUMN     "email" TEXT,
+ADD COLUMN     "organizationId" TEXT NOT NULL;
+
+-- AlterTable
+ALTER TABLE "Organization" DROP COLUMN "foundation_date",
+DROP COLUMN "long_description",
+DROP COLUMN "number_of_users",
+DROP COLUMN "owner_id",
+ADD COLUMN     "foundationDate" TIMESTAMP(3) NOT NULL,
+ADD COLUMN     "longDescription" TEXT NOT NULL,
+ADD COLUMN     "numberOfUsers" INTEGER NOT NULL,
+ADD COLUMN     "ownerId" TEXT NOT NULL,
+ADD COLUMN     "slug" TEXT NOT NULL;
+
+-- AlterTable
+ALTER TABLE "Projects" DROP COLUMN "organization_id",
+ADD COLUMN     "organizationId" TEXT NOT NULL;
+
+-- CreateTable
+CREATE TABLE "Tag" (
+    "id" TEXT NOT NULL,
+    "text" TEXT NOT NULL,
+
+    CONSTRAINT "Tag_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "_OrganizationToTag" (
+    "A" TEXT NOT NULL,
+    "B" TEXT NOT NULL
```

Comment:
What's this?

Reply:
I don't understand this A and B scheme :/

Reply:
those are automatically generated migration, i don't write them by myself

in short, this is how prisma handles many-to-many relationships by making implicit middleman table